From 436839c17c999e6a2833c3f8ca7166d7612c53fc Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 20 Oct 2020 16:11:53 +0300
Subject: [PATCH] [SW-12754] gaudi: security: remove done todos

All the static security has been implemented, so there is no more need
to have both of TODOs.
Remove them.

Change-Id: I9fa43b9b2041e01f135aabf10c1bd98b1a08f84e
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/security.c b/gaudi/security.c
index 9cf5490..be9e164 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -757,8 +757,6 @@
 
 	/* Protect GLOBAL_CONF - 4K */
 	preboot_secure_psoc_global_conf();
-
-	/* TODO: handle PSOC_DBG/PSOC_ETR/AXICTL */
 }
 
 static void preboot_secure_pci_wrap(void)
@@ -1327,8 +1325,6 @@
 
 	preboot_secure_lbw();
 	preboot_secure_hbw();
-
-	/* TODO: implement the rest of security enablement */
 
 	cpu_boot_err_clrset(CPU_BOOT_ERR0_SECURITY_NOT_RDY, false);
 	preboot_puts("Security init finished.\n");
